node/src/main.rs

```diff
@@ -74,34 +78,36 @@ async fn main() -> Result<()> {
     };
 
     // In benchmarks, transactions are not deserializable => many errors at the debug level
+    // Moreover, we need RFC 3339 timestamps to parse properly => we use a custom subscriber.
     cfg_if::cfg_if! {
         if #[cfg(feature = "benchmark")] {
             let custom_directive = "executor::core=info";
-        } else {
-            let custom_directive = "";
-        }
-    }
+            let filter = EnvFilter::builder()
+            .with_default_directive(LevelFilter::INFO.into())
+            .parse(format!(
+                "{tracing_level},h2={network_tracing_level},tower={network_tracing_level},hyper={network_tracing_level},tonic::transport={network_tracing_level},{custom_directive}"
+            ))?;
 
-    let filter = EnvFilter::builder()
-        .with_default_directive(LevelFilter::INFO.into())
-        .parse(format!(
-            "{tracing_level},h2={network_tracing_level},tower={network_tracing_level},hyper={network_tracing_level},tonic::transport={network_tracing_level},{custom_directive}"
-        ))?;
+            let env_filter = EnvFilter::try_from_default_env().unwrap_or(filter);
 
-    let env_filter = EnvFilter::try_from_default_env().unwrap_or(filter);
-    cfg_if::cfg_if! {
-        if #[cfg(feature = "benchmark")] {
             let timer = tracing_subscriber::fmt::time::UtcTime::rfc_3339();
             let subscriber_builder = tracing_subscriber::fmt::Subscriber::builder()
-                .with_env_filter(env_filter)
-                .with_timer(timer).with_ansi(false);
+            .with_env_filter(env_filter)
```

Comment:
I feel like we should be able to easily adapt TelemetrySubscribers to accommodate this benchmarking use case as well.  Maybe just add a flag to turn off ansi or whatever else is needed.